library/styleguideComponents/ReactComponent/Utils/styles.css

```diff
@@ -94,6 +94,31 @@
     color: #576574;
     align-self: flex-start;
     text-align: justify;
+    overflow: hidden;
+    max-height: 4rem;
+}
+
+.react-rainbow-utils-item_description::before {
+    content:'...';
+    position: absolute;
+    top: 10.6875rem;
+    right: 0.375rem;
```

Comment:
with this solution you are always set the ellipsis and we need that when the description is short and fit in 3 lines then should not have ellipsis

Reply:
sorry I saw that this works